wcfsetup/install/files/lib/acp/form/CaptchaQuestionEditForm.class.php

```diff
@@ -23,119 +21,21 @@ class CaptchaQuestionEditForm extends CaptchaQuestionAddForm
      */
     public $activeMenuItem = 'wcf.acp.menu.link.captcha.question.list';
 
-    /**
-     * edited captcha question
-     * @var CaptchaQuestion
-     */
-    public $captchaQuestion;
-
-    /**
-     * id of the edited captcha question
-     * @var int
-     */
-    public $captchaQuestionID = 0;
-
     /**
      * @inheritDoc
      */
-    public function assignVariables()
-    {
-        parent::assignVariables();
-
-        I18nHandler::getInstance()->assignVariables(!empty($_POST));
+    public $formAction = 'edit';
 
-        WCF::getTPL()->assign([
-            'action' => 'edit',
-            'captchaQuestion' => $this->captchaQuestion,
-        ]);
-    }
-
-    /**
-     * @inheritDoc
-     */
-    public function readData()
+    public function readFormParameters()
     {
-        parent::readData();
+        parent::readFormParameters();
 
-        if (empty($_POST)) {
-            I18nHandler::getInstance()->setOptions(
-                'question',
-                1,
-                $this->captchaQuestion->question,
-                'wcf.captcha.question.question.question\d+'
-            );
-            I18nHandler::getInstance()->setOptions(
-                'answers',
-                1,
-                $this->captchaQuestion->answers,
-                'wcf.captcha.question.question.answers\d+'
-            );
-
-            $this->isDisabled = $this->captchaQuestion->isDisabled;
+        if (\isset($_REQUEST['id']) && \is_numeric($_REQUEST['id'])) {
```

Comment:
```suggestion
        if (isset($_REQUEST['id']) && \is_numeric($_REQUEST['id'])) {
```

`isset()` is not a function.